corehq/apps/api/resources/v0_4.py

```diff
@@ -120,6 +120,7 @@ def obj_get_list(self, bundle, domain, **kwargs):
 
         def wrapper(doc):
             if doc['doc_type'] in xform_doc_types:
+                doc.pop('user_id', None)  # needed when xform is stored in sql
                 return xform_doc_types[doc['doc_type']].wrap(doc)
```

Comment:
@benrudolph I think the `XFormInstanceSQLSerializer` intentionally sets `'doc_type'` to `'XFormInstance'` so that couch and sql are consistent so I don't think that's happening. I still don't get why we would need to wrap the json in the APIs though. Guessing just wanting to take advantage of some computed properties which we could maybe get rid of by making the serialization from sql/couch more explicit.